roles/metrics-export.lua

```diff
@@ -258,25 +298,47 @@ local function apply_http(conf)
     local enabled = {}
     for _, node in ipairs(conf) do
         if #(node.endpoints or {}) > 0 then
-            local host, port, err = parse_listen(node.listen)
-            if err ~= nil then
-                error("failed to parse URI: " .. err, 2)
+            local host, port, target
+            if node.server ~= nil then
+                local server = httpd_role.get_server(node.server)
+                host, port = server.host, server.port
```

Comment:
Why do we need host and port values in the case? It seems useless.

Reply:
Yes, my bad, thank you. Fixed.